### PR TITLE
Use Null Coalesce Operator when possible

### PR DESCRIPTION
**Pull request by carusogabriel**
Just a small refactoring to use the Null Coalesce Operator when possible 💅

### CODE REVIEW

File: src/Http/ServerRequest.php [hunk 1]
```diff
@@ -1745,11 +1745,8 @@ public function getAttribute($name, $default = null)
         if (in_array($name, $this->emulatedAttributes, true)) {
             return $this->{$name};
         }
-        if (array_key_exists($name, $this->attributes)) {
-            return $this->attributes[$name];
-        }
 
-        return $default;
+        return $this->attributes[$name] ?? $default;
```
Comment (carusogabriel) [inline src/Http/ServerRequest.php]: Should we keep the Early Returns in this case?

**Comment by ADmad** [comment]
Your changes will yield different results if an array key is set to null. That's the reason `array_key_exits()` check is used.

**Comment by dereuromark** [comment]
array_key_exists() int the above cases cannot be replaced as ADmad pointed out.
In cases where null can be a valid value neither isset() or ?? are valid operators as they now change behavior, returning not the desired result.

**Comment by dereuromark** [comment]
That said, I find it alarming how many tests actually passed here :D

**Comment by ADmad** [comment]
@dereuromark Yeah, we do need more tests for keys with null values :blush:.

**Pull request closed without merging**